packages/iov-keycontrol/src/proposal.ts

```diff
@@ -74,15 +74,14 @@ export interface HDKeyringEntry extends KeyringEntry<LocalHDIdentity> {
   // a keyring entry may support many possible paths, or only one.
   // if path or algorithm is not supported by the implementation,
   // it will throw an error
-  //
-  // if autoincrement is true, it will append an auto-incrementing counter
-  // to the path as first unused space.
-  // eg. if path is 44'/13'/0'/0 and auto-increment is true, it will look
-  // see 44'/13'/0'/0/{0,1,2,3} all exist and create with path 44'/13'/0'/0/4
   readonly createIdentityWithPathPrefix: (
     algo: Algorithm,
-    path: ReadonlyArray<Slip0010RawIndex>,
-    autoincrement: boolean,
+    // purpose should be hard-coded as 44' ?
```

Comment:
Yes.

Reply:
We can default this in functions before calling it, though. As we probably need to be flexible in the event that something changes for purpose in the future